README.md

```diff
@@ -1,25 +1,66 @@
+![Latest release](https://img.shields.io/github/release/common-workflow-language/common-workflow-language.svg?style=flat)
+[![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/common-workflow-language/common-workflow-language?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+[![GitHub stars](https://img.shields.io/github/stars/common-workflow-language/common-workflow-language.svg)](https://github.com/common-workflow-language/common-workflow-language/stargazers)
+
 Common Workflow Language
 ========================
 
-[**Support**](#Support) [![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/common-workflow-language/common-workflow-language?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
-[![GitHub stars](https://img.shields.io/github/stars/common-workflow-language/common-workflow-language.svg)](https://github.com/common-workflow-language/common-workflow-language/stargazers)
+* Describe analysis workflows and tools of arbitrary complexity using plain text
```

Comment:
I like putting the strengths up front, I think this section needs a little more framing.  It is implicitly answering the question "what is CWL" or "what can I do with CWL" and I think explicitly posing that question would make it read a little better.